docs/decisions/0002-footer-links.rst

```diff
@@ -0,0 +1,168 @@
+2. Footer Links
+########################
+.. The title should be a short noun phrase. For example, "1. Django IDA" or "9. LDAP for Multitenant Integration"; filename should be lowercase with punctuation removed and spaces replaced by dash
+
+Status
+******
+
+**Draft (=> Accepted)**
+
+.. **Accepted** *2023-05-11*
+
+.. Standard statuses
+    - **Draft** if the decision is newly proposed and in active discussion
+    - **Provisional** if the decision is still preliminary and in experimental phase
+    - **Accepted** *(date)* once it is agreed upon
+    - **Superseded** *(date)* with a reference to its replacement if a later ADR changes or reverses the decision
+
+    If an ADR has Draft status and the PR is under review, you can either use the intended final status (e.g. Provisional, Accepted, etc.), or you can clarify both the current and intended status using something like the following: "Draft (=> Provisional)". Either of these options is especially useful if the merged status is not intended to be Accepted.
+
+Context
+*******
+
+.. This section describes the forces at play, including technological, political, social, and project local. These forces are probably in tension, and should be called out as such. The language in this section is value-neutral. It is simply describing facts.
+
+The footer component is basic - it only has a trademark logo and a language
+selector. It doesn't look similar to the legacy UI footer.
+
+It also doesn't have many customization options. You can customize
+LMS base URL and trade mark logo.
+
+If you want to have more advanced customizations, it's necessary to fork the
+component, modify it, and then install the fork as a package instead of the
+default footer at build time of MFEs.
+
+We want to make the MFE footer to closer resemble the legacy UI footer by
+adding the same footer links as in the legacy UI, and to make it easier to
+style the footer component.
+
+Decision
+********
+
+.. This section describes our response to these forces. It is stated in full sentences, with active voice. "We will …"
+
+Links
+=====
+
+For specifying the footer links, we are going to use `MFE's config mechanism`_.
+That would allow to specify a list of links in various ways:
+
+* At build time:
+  * Via environmental variables
```

Comment:
Will this be possible? Does this mean having a large environment variable with a JSON string? I'm not sure if that works. It might though, or at least there should be a way to make it work.

Reply:
@xitij2000 [config.js](https://github.com/openedx/frontend-platform/blob/master/src/config.js#L25-L39) does check the env vars. However, you're right that env vars store strings, not js arrays or object, and since we represent the `FOOTER_LINKS` as a list of objects, without additional parsing (which is not implemented), it wouldn't work. I'll remove this.